raiko-guests/sgx/src/one_shot.rs

```diff
@@ -108,15 +103,7 @@ pub async fn one_shot(global_opts: GlobalOpts, args: OneShotArgs) -> Result<()>
     println!("Global options: {global_opts:?}, OneShot options: {args:?}");
 
     let path_str = args.blocks_data_file.to_string_lossy().to_string();
-    let block_no = u64::from_str(&String::from(
-        args.blocks_data_file
-            .file_prefix()
-            .unwrap()
-            .to_str()
-            .unwrap(),
-    ))?;
-
-    println!("Reading input file {path_str} (block no: {block_no})");
+    println!("Reading input file {path_str}");
 
     let privkey_path = global_opts.secrets_dir.join(PRIV_KEY_FILENAME);
```

Comment:
Can we don't do too much path joining? Just pass in the entire path or hardcode it as a const

Reply:
There's some possibilities here with how files are mapped in SGX so I think we could make the path very simple and basically the file itself so most of it could be handled externally. But even just having everything in the same place for all file paths may already be cleaner and easier.

After the real SGX support is confirmed working as well (and also with docker), we can see what is possible.